### ISSUE TITLE
Bubbles display order for a same page

### ISSUE THREAD

**Issue by MaartenLMEM** [dis-moi/extension#851]
**User issues**
- I don"t know why a bubble is diplayed before another
- I would like to see most relevant bubbles as first

**Describe the feature/improvement**
Different options to define the ordre : 
1. Display as first last modified bubble, and so on.
2. Display bubble as a function of there open rate (click on detail / displayed) : highest rate is diplayed as first. 
I prefer second option.
But in this case we need to think a special case : what s default rate a new bubble that have never been displayed ? 10% for the purpose of displaying this new one before a bubble with a very bad open rate.

**Target user**
Users who cross pages with mutiples bubbles.

On the same subject : https://github.com/dis-moi/extension/issues/850

**Comment by christpet** [comment]
Good suggestion, thanks @MaartenLMEM . Let's keep this in the backlog until we are clear on the impact it will have.

**Comment by lutangar** [comment]
No matter what we choose to implement here, the algorithm behind this "content prioritization algorithm" (editorialization?! cc @christpet is that a real word? ) should be publicly documented in plain english. 
These "rules" should be public knowledge.

**Comment by christpet** [comment]
@lutangar that's a good point, a good +1 for transparency.
